apps/admission-webhook-k8s/mutating-webhook-config.yaml

```diff
@@ -0,0 +1,48 @@
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  name: mutating-webhook
+  labels:
+    spiffe.io/webhook: "true"
+webhooks:
+- admissionReviewVersions:
+  - v1
+  clientConfig:
+    service:
+      name: admission-webhook-svc
+      namespace: nsm-system
+      path: /mutate
+      port: 443
+  failurePolicy: Fail
+  matchPolicy: Equivalent
+  name: nsm-admission-webhook-k8s-5dfd78487d-26n5b.networkservicemesh.io
```

Comment:
This looks like a generated pod name.
We can probably use a more generic name, like `nsm-admission-webhook-k8s.networkservicemesh.io`.

Reply:
Thanks, fixed